Components/Transform/Drivers/MultiValueArithmeticDriver.cs

```diff
@@ -1,70 +1,69 @@
 ﻿using System.Linq;
-using Elements.Assets;
 using Elements.Core;
 using FrooxEngine;
 
 namespace Obsidian;
 
+public enum ArithmeticMode
+{
+	Addition,
+	Subtraction,
+	Multiplication,
+	Division
+}
+
 [Category(new string[] { "Obsidian/Transform/Drivers" })]
 [GenericTypes(GenericTypesAttribute.Group.Primitives)]
 public class MultiValueArithmeticDriver<T> : Component
 {
-    public static bool IsValidGenericType => Coder<T>.SupportsAddSub;
-
-    public enum ArithmeticMode
-    {
-        Addition,
-        Subtraction,
-        Multiplication,
-        Division
-    }
+	public static bool IsValidGenericType => Coder<T>.SupportsAddSub;
 
-    public readonly FieldDrive<T> Target;
+	public readonly FieldDrive<T> Target;
 
-    public readonly Sync<ArithmeticMode> Mode;
+	public readonly Sync<ArithmeticMode> Mode;
 
-    public readonly SyncList<Sync<T>> Values;
+	public readonly SyncList<Sync<T>> Values;
 
-    protected override void OnChanges()
```

Comment:
we should add a workflow to do this for us when files are submitted like the old project